src/main/scala/org/clulab/habitus/variables/VariableProcessor.scala

```diff
@@ -36,8 +37,7 @@ class VariableProcessor(val processor: CluProcessor,
   def parse(doc: Document): ParsingResult = {
     // extract mentions from annotated document
     val mentions = extractor.extractFrom(doc).sortBy(m => (m.sentence, m.getClass.getSimpleName))
-
-    val (tbms, relsAndEvents) = mentions.partition(_.isInstanceOf[TextBoundMention])
+    val (tbms, _) = mentions.partition(_.isInstanceOf[TextBoundMention])
```

Comment:
Since the second partition is being thrown away, is this
`val tbms = mentions.filter(_.isInstanceOf[TextBoundMention])`
or
`val tbms = mentions.collect(mention: TextBoundMention => mention }`